src/compounds/ad-banner/src/index.tsx

```diff
@@ -22,15 +22,17 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
   href: string
   imageCritical?: boolean
   className?: string
+  targetAttribute?: string
+  uspColor?: string
 }
 
 const AdBanner: React.FC<Props> = ({
   title,
   content,
   usp,
   sponsor,
-  backgroundColor = 'black',
-  textColor = 'white',
+  backgroundColor = '',
```

Comment:
if you're doing backgroundColor || black in a few places, shouldn't this just be backgroundColor = 'black' to simplify that?